universal-application-tool-0.0.1/app/services/applicant/question/Scalar.java

```diff
@@ -41,6 +41,42 @@
   UPDATED_AT,
   PROGRAM_UPDATED_IN;
 
+  @Override
+  public String toString() {
```

Comment:
question: Thoughts on making this a `asHumanReadableString` method rather than overriding? Not sure it matters too much, but might be helpful for clarity

Reply:
Let me know what you think of the new `Scalar` implementation.